packages/grafana-data/src/dataframe/utils.ts

```diff
@@ -12,6 +12,28 @@ export function isTimeSeriesFrame(frame: DataFrame) {
   // and at least one number field
   const timeField = frame.fields.find((field) => field.type === FieldType.time);
   const numberField = frame.fields.find((field) => field.type === FieldType.number);
+
+  // There are certain query types in which we will
+  // get times but they will be the same or not be
+  // in increasing order. To have a time-series the
+  // times need to be ordered from past to present
+  if (timeField !== undefined) {
```

Comment:
It's mention in description that there might be multiple time fields, but this test only checks the first one. Is it always guranteed that the relevant time field is always first?
I'd also expect tests to reflect this scenario, with multiple time fields :)

Reply:
I guess we could change the order of this function a bit. Instead of trying to find the time field based on the type we could also add the time comparison check to that find call.

Basically timeField would be first available field with the type time and has the correct sorting.

Reply:
I updated the ordering, but also the logic. We check to see that any configured field is in time series format before presenting to the user along with making sure that the frame contains any time field at all.